### PR TITLE
Extracted all encoding and put them in to a series

### PR DESCRIPTION
**Pull request by Ahmed-Mohiuddin-Shah**
Extracted all encoding and put them in to a series with proper primary keys as indexes

### CODE REVIEW

File: attendance.py [hunk 1]
```diff
@@ -9,4 +13,30 @@ def main():
...
+        with open(f"./{directoryName}/{filename}") as f:
+            reader = csv.reader(f)
+            encodings = []
+            for code in reader:
+                encodings.extend(code)
+            encodings = [float(code) for code in encodings]
+            return encodings
+
+    knownEncodingFiles = os.listdir(f"./{directoryName}")               #Getting names of all csv files in directory
+    encodings=[]                                                        #variable definition
```
Comment (xwhiz) [inline attendance.py]: You should make it an empty Series like  encodings = pd.Series(dtype='float64')

File: attendance.py [hunk 2]
```diff
@@ -9,4 +13,30 @@ def main():
...
+    encodingPrimaryKey=[]                                               #variable definition
```
Comment (xwhiz) [inline attendance.py]: Then we don't need this when we initialize it as Series

File: attendance.py [hunk 3]
```diff
@@ -9,4 +13,30 @@ def main():
...
+    for file in knownEncodingFiles:                                     #Looping over the files to extract encodings
+        encodings.append(getCSV(file))                                  #Saving encodings in encodins list
```
Comment (xwhiz) [inline attendance.py]: instead of this, we use this syntax which does both of these in one step, encodings[file[:-3]] = getCSV(file)

**Review by xwhiz — COMMENTED** [review]
Make these changes accordingly

File: attendance.py (hunk 3 above)
Comment (Ahmed-Mohiuddin-Shah) [reply attendance.py]: My code is easier on the eyes but whatever you say big boss.

Comment (xwhiz) [reply attendance.py]: Yeah it is really good but we should use the abstraction that library provides us...

**Comment by Ahmed-Mohiuddin-Shah** [comment]
Changes made BigBoss

**Pull request merged**